docs/content/en/usage/features.md

```diff
@@ -674,3 +674,12 @@ feedback.internal.google.com (endpoint) -> [ team:human resources, public_facing
 Endpoint Meta Importer can be found in the Endpoint tab when viewing a Product
 
 **Note:** The field "hostname" is required as it is used to query/create endpoints.
+
+## Findings Image Upload
+
+You can add images (.png) to your findings. In order to achieve this, you have to click on "Manage Files" within the finding:
```

Comment:
It looks like `.jpeg` [will work by default](https://github.com/DefectDojo/django-DefectDojo/blob/d698a7a1ff2a914754a541140b236dd3092d7e8e/dojo/settings/settings.dist.py#L261) as well. Not sure why we left off e.g. `.gif` in the default setting, but I confirmed that adding `.gif` to `DD_FILE_UPLOAD_TYPES` will work as well.